src/lib/controllers/ui-controller.js

```diff
@@ -589,7 +589,14 @@ export default class UIController {
         },
 
         uiOptionChange: function (name, value) {
-          if (name === 'fontSize' || name === 'colorSchema') { this.uiController.uiOptions.items[name].setValue(value) } else { this.uiController.uiOptions.items[name].setTextValue(value) }
+          // TODO this should really be handled within OptionsItem
```

Comment:
I am using here setValue instead of setTextValue because it is an inner property and I am handling with it only with the help of value, and not using text.

Reply:
Yes I understand this. I believe it exposes a need for refactoring of Options as described at #73